src/main/java/ai/lucidtech/las/sdk/Credentials.java

```diff
@@ -75,7 +76,40 @@ public String getApiEndpoint() {
         return apiEndpoint;
     }
 
-    private Map readFromEnviron() {
+    private void setCredentials(String credentialsPath) {
+        Map credentialsFromEnv = this.readFromEnv();
+        Map credentialsFromFile = this.readFromFile(credentialsPath);
+        Map<String, String> credentials;
+
+        System.out.println("env: " + credentialsFromEnv);
+        System.out.println("file: " + credentialsFromFile);
+
+        if (all(credentialsFromEnv)) {
+            credentials = credentialsFromEnv;
+        } else if (all(credentialsFromFile)) {
+            credentials = credentialsFromFile;
+        } else {
+            return;
```

Comment:
Can be throw an exception instead of ending up with Credentials that have null values?